fix(tools): adding process.env.NODE_ENV replacement to dev-server.ts (#1942)
patternfly/patternfly-elements: .changeset/rotten-cups-roll.md

---
"@patternfly/pfe-tools": minor
---

> Adding @rollup/plugin-replace to dev server to fix issue with undefined process.env.NODE_ENV

Reviewer: ```suggestion
Configures dev server to automatically replace instances of `process.env.NODE_ENV` with `production` in library code.
```

> Configures dev server to automatically replace instances of `process.env.NODE_ENV` with `production` in library code.

Reviewer: ```suggestion
Automatically replaces `process.env.NODE_ENV` with `production` both in dev server and production bundles.
```
